contrib_bots/lib/yoda_bot.py

```diff
@@ -0,0 +1,129 @@
+# See readme-yoda-bot.md for instructions on running this code.
+from __future__ import print_function
+import unirest
+import os
+import logging
+import ssl
+
+
+class ServerError(ssl.SSLError):
+    '''raise this when there is an error with the server'''
+
+
+class ApiKeyError(Exception):
+    '''raise this when there is an error with the Mashape Api Key'''
+
+
+class YodaSpeakHandler(object):
+    '''
+    This bot will allow users to translate a sentence into 'Yoda speak'.
+    It looks for messages starting with '@yoda'.
+    '''
+
+    def usage(self):
+        return '''
+            This bot will allow users to translate a sentence into
+            'Yoda speak'.
+            Users should preface messages with '@yoda'.
+
+            Before running this, make sure to get a Mashape Api token.
+            Instructions are in the 'readme-yoda-bot.md' file.
+            Store it in the 'yoda_api_key.txt' file.
+            The 'yoda_api_key.txt' file should be located at '~/yoda_api_key.txt'.
+            Example input:
+            @yoda You will learn how to speak like me someday.
+            '''
+
+    def triage_message(self, message):
+        # return True iff we want to (possibly) response to this message
+
+        original_content = message['content']
+
+        is_sentence = (original_content.startswith('@yoda'))
```

Comment:
No need for brackets. `return original_content.startswith('@yoda')` should also work.